### DISCUSSION TITLE
Limit number of rules/objects affected by action

### DISCUSSION THREAD

**Discussion by bethatasitmay** [PaloAltoNetworks/pan-os-php#683] [Q&A] [answered]
Hi

Is it currently possible to limit the number of rules or objects that are affected by an action?

Let's say I want to add a tag to rules that match a filter, but I only need to tag the first X number of rules or objects (where X  is a positive integer).

For example, let's say I have 1,000 rules that match a filter, but I don't want to work with them all at once.  So, I only want to tag the first 100 of them.

For my purposes, I'm actually going to be applying two or more tags as I'm tagging rules disabled & marking them for future deletion and disabling the rules (the rule disable can be done separately).  Future runs of the command could be further filtered by the extra tag(s).  For various reasons, I don't want to handle all of the rules that match a filter all at once.

At the moment, I'm only interested in rules, but I might want to do this for objects as well.

**Comment by swaschkut** [comment, marked as answer]
this is not implemented, and will also not be implemented in a short time frame.
you can still create a FeatureRequest for this.
